addons/test_discuss_full/tests/test_performance.py

```diff
@@ -20,11 +20,13 @@ class TestDiscussFullPerformance(HttpCase):
     #     6: odoobot format
     #     4: settings
     #     1: has_access_livechat
+    #     3: _compute_is_editable
```

Comment:
This is not for `_query_count_init_store` but for `_query_count` (comment for that counter should be added and moved down)